packages/scandipwa/src/component/ProductPrice/ProductPrice.component.js

```diff
@@ -29,6 +29,7 @@ export class ProductPrice extends PureComponent {
         priceType: PropTypes.oneOf(Object.values(PRODUCT_TYPE)),
         originalPrice: PropTypes.object,
         tierPrice: PropTypes.string,
+        configuration: PropTypes.object,
```

Comment:
`PropTypes.object` is a forbidden type in our eslint config but right now this rule is disabled since it will require a lot of fixed in the code.
Still to make our life easier please use `PropTypes.share({})`.